target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl839x-sram.S

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Realtek RTL839X SRAM clock setter
+ * Copyright (C) 2022 Markus Stockhausen
+ * 
+ * This code changes the clocks of a RTL839x SOC. It has been isolated
+ * from the rest of the module to be relocatable to SRAM. We avoid any
+ * normal stack or memory accesses. So we can change PLL rates in a
+ * safe way.
+ */
+
+#include <dt-bindings/clock/rtl83xx-clk.h>
+
+#define RTL838X_SW_BASE		0xbb000000
```

Comment:
`838X_BASE`, but the file is 839x-related.

Reply:
fixed in upcoming version